### ISSUE TITLE
bug: `severity` config is ignored

### ISSUE THREAD

**Issue by chrisgrieser** [rvben/rumdl#217]
According to [RULES.md](https://github.com/rvben/rumdl/blob/main/docs/RULES.md#rule-severities), the diagnostic severity of a rule is configurable via `severity`:

```toml
[MD004]  # ul-style
style = "dash"
severity = "info"
```

However, the LSP seems to ignore it (the diagnostic is still a warning), and running `rumdl fmt` gives this warning:

```
[config warning] Unknown option for rule MD004: severity
```

**Comment by rvben** [comment]
Thanks for reporting this! In 0.0.200, this works:

```
[MD007]
severity = "error"   # Upgrades warning → error

[MD042]
severity = "warning" # Downgrades error → warning
```

This is reflected in SARIF output ("level": "error" vs "level": "warning").

**Comment by chrisgrieser** [comment]
Thanks!

Would it be possible to actually add "info" as severity level? 

I actually like having more differentiation options, for example, I use error for critical issues, info for issues that formatting will fix, and warning for the rest. That way I can configure my editor to treat those three levels differently. 

Having three severities options instead of two gives you more differentiation potential to work with, while having no downside I can think of.

**Comment by rvben** [comment]
Thanks for the feedback. I don't have the use-case clear, but if this is something that helps and is low-risk, it might be fine to add.

**Comment by rvben** [comment]
The 'info' severity will be added in upcoming release. 👍

**Comment by chrisgrieser** [comment]
Thank you!

**Comment by chrisgrieser** [comment]
> I don't have the use-case clear

Just to give you an example. In some cases, diagnostics can be really noisy, like with some misaligned table cells.

<img width="2744" height="1650" alt="Image" src="https://github.com/user-attachments/assets/19a74061-de55-4e72-9999-c321ea2fec91" />

To deal with that, I have configured my editor to hide the diagnostics and only display the affected lines when the severity is `info`, resulting in this:

<img width="2650" height="928" alt="Image" src="https://github.com/user-attachments/assets/f0a9bbc7-9dd7-4a53-92ee-fca670556bba" />

With only warnings and errors as severities, I'd be forced to configure my editor to hide warnings, and configure every single LSP and linter I use to output all the stuff I want to see as `error` to achieve the above. With `info` as severity being available, I just set `MD060` to `info` and I'm done.

**Comment by rvben** [comment]
Thanks @chrisgrieser for the clear example. 👍
